servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/KeepAliveManager.java

```diff
@@ -65,8 +70,17 @@ private enum State {
     private static final Logger LOGGER = LoggerFactory.getLogger(KeepAliveManager.class);
     private static final AtomicIntegerFieldUpdater<KeepAliveManager> activeStreamsUpdater =
             AtomicIntegerFieldUpdater.newUpdater(KeepAliveManager.class, "activeStreams");
-    private static final long GRACEFUL_CLOSE_PING_CONTENT = ThreadLocalRandom.current().nextLong();
-    private static final long KEEP_ALIVE_PING_CONTENT = ThreadLocalRandom.current().nextLong();
+
+    // Use the last digit (even or odd) to distinguish PING frames when frame logging is enabled.
+    private static final long GRACEFUL_CLOSE_PING_CONTENT = ThreadLocalRandom.current().nextLong() | 0x01L;
+    private static final long KEEP_ALIVE_PING_CONTENT = ThreadLocalRandom.current().nextLong() & 0xfffffffffffffffeL;
```

Comment:
nit: would `^ 0x01` result in the same and be easier to read?

Reply:
it wouldn't ignore that.

Reply:
Nope, XOR won't work here because if the random bit was set to 0, it will flip it to 1.

Reply:
I will do `& ~0x01L`